src/actions/notificationActions.js

```diff
@@ -1,21 +1,24 @@
 import * as types from '../actions-types';
-import { axiosHelper } from '../helpers/index';
+import { apiAction } from '../helpers/index';
 
-export const createOne = config => async (dispatch) => {
-  try {
-    const res = await axiosHelper().post('/notifications/configuration', { config });
-    const { data } = res;
-    dispatch({
-      type: types.notificationActionTypes.CREATE_ONE,
-      payload: data
-    });
-  } catch (error) {
-    dispatch({
-      type: types.notificationActionTypes.CATCH_ERROR,
-      payload: error.response.data.errors
-    });
-  }
-};
+export const createOne = config => async dispatch => dispatch(apiAction({
```

Comment:
Are you using the rest helper function introduced earlier?

Reply:
Yes, I am using apiAction helper